### PR TITLE
Build script also updates the vsix manifest version number.

### PR DESCRIPTION
**Pull request by TWith2Sugars**

### CODE REVIEW

File: build.fsx [hunk 1]
```diff
@@ -67,8 +69,8 @@ Target "AssemblyInfo" (fun _ ->
   let shared =
       [ Attribute.Product project
         Attribute.Description summary
-        Attribute.Version release.AssemblyVersion
-        Attribute.FileVersion release.AssemblyVersion ] 
+        Attribute.Version version
```
Comment (dungpa) [inline build.fsx]: I think it's better to keep `release.AssemblyVersion` here. Assembly version doesn't have any effect in installing the extension.

File: build.fsx [hunk 2]
```diff
@@ -86,6 +88,17 @@ Target "AssemblyInfo" (fun _ ->
       (Attribute.InternalsVisibleTo "FSharpVSPowerTools.Tests" :: Attribute.Title "FSharpVSPowerTools.Logic.VS2015" :: shared) 
 )
 
+Target "VsixManifest" (fun _ ->
+    let manifest = "./src/FSharpVSPowerTools/source.extension.vsixmanifest"
+    let doc = new XmlDocument() in
```
Comment (dungpa) [inline build.fsx]: Maybe we should use `XmlDocument(PreserveWhitespace=true)`?

File: build.fsx [hunk 3]
```diff
@@ -301,6 +314,7 @@ Target "All" DoNothing
 "Clean"
   =?> ("BuildVersion", isAppVeyorBuild)
   ==> "AssemblyInfo"
+  ==> "VsixManifest"
```
Comment (dungpa) [inline build.fsx]: We should only run this on AppVeyorCI. Locally, we don't want FAKE build to override our vsix version.

I think a better way is to extract Version value (e.g. 2.1.3) and append build numbers into it?

**Comment by TWith2Sugars** [comment]
Hows that?

**Comment by dungpa** [comment]
One last thing:

```
node.Attributes.GetNamedItem("Version").Value <- version
```

should be

```
let currentVersion = node.Attributes.GetNamedItem("Version").Value
node.Attributes.GetNamedItem("Version").Value <- sprintf "%s.%s" currentVersion AppVeyor.AppVeyorEnvironment.BuildNumber
```

As it is, users have to uninstall builds from AppVeyor when the official one is released.

**Comment by TWith2Sugars** [comment]
Would they? The build number for the released on would also be incremented.

Either way its done.

**Comment by dungpa** [comment]
@FeodorFitsner Is there any bug on AppVeyor deployment? We see builds from other users and some pushes are not built at all.

**Comment by FeodorFitsner** [comment]
Looking in that. Can you drop a link to a build please?



- Feodor

**Comment by TWith2Sugars** [comment]
https://ci.appveyor.com/project/dungpa/visualfsharppowertools/history

**Comment by TWith2Sugars** [comment]
and https://ci.appveyor.com/project/dungpa/visualfsharppowertools/build/1.0.1753

**Comment by FeodorFitsner** [comment]
Cancel queued builds and start new one - it should be fine now. Please let me know about the results.

**Comment by dungpa** [comment]
@FeodorFitsner Everything is ok now.

**Comment by dungpa** [comment]
@TWith2Sugars Great, thanks.

**Pull request merged**